src/partials/_about.hbs

```diff
@@ -0,0 +1,7 @@
+<section class="about">
+	<h2 class="about__heading">About</h2>
+
+	<p class="about__text">A Modern Eden is a line of products designed for style savvy parents who care about what they put in their kids’ rooms, prints, totes, accessories, and mobile apps that feature mid-century modern animal illustrations. Developed by creative agency Sparkbox, located in Dayton, Ohio, A Modern Eden products feature sustainable materials and hand-crafted goodness.</p>
+
+	<a href="https://seesparkbox.com/" class="about__logo icon-sparkbox-logo">Sparkbox</a>
+</section>
```

Comment:
This red crossed-out circle and return arrow mean this file doesn't end in a new line. It's a best practice to end every file with at least one new line. In some editors you can actually configure it to perform these best practice whitespace settings every time you save the file.
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline